tests/random/test_random_func_return_range.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+
+import pytest
+
+from generic_grader.random.random_func_return_range import build
+from generic_grader.utils.options import Options
+
+
+@pytest.fixture()
+def built_class():
+    """Provide the class built by the build function."""
+    return build(Options())
+
+
+@pytest.fixture()
+def built_instance(built_class):
+    """Provide an instance of the built class."""
+    return built_class()
+
+
+def test_random_func_return_range_build_class(built_class):
+    """Test that the style comments build function returns a class."""
+    assert issubclass(built_class, unittest.TestCase)
+
+
+def test_random_func_return_range_build_class_name(built_class):
+    """Test that the built_class has the correct name."""
+    assert built_class.__name__ == "TestRandomFuncReturnRange"
+
+
+def test_random_func_return_range_built_instance_type(built_instance):
+    """Test that the built_class returns instances of unittest.TestCase."""
+    assert isinstance(built_instance, unittest.TestCase)
+
+
+def test_random_func_return_range_has_test_method(built_instance):
+    """Test that instances of the built_class have test method."""
+    assert hasattr(built_instance, "test_random_func_return_range_0")
+
+
+# Cases Tested:
+# 1. Correct returned value
+# 2. Expected set is larger than the returned set
+# 3. Returned set is larger than the expected set
+
+
+cases = [
+    {  # Correct returned value
+        "submission": "import random as r\ndef test_function():\n    return r.randint(1, 10)",
+        "reference": "import random as r\ndef test_function():\n    return r.randint(1, 10)",
```

Comment:
These reference modules don't actually do anything.  Let's just set them to `pass`.